### PR TITLE
[ Application ] Doc for Logistic Regression

### PR DESCRIPTION
**Pull request by jijoongmoon**
Add doc for logistic regression application.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: Applications/LogisticRegression/README.md [hunk 1]
```diff
@@ -1,2 +1,71 @@
...
+5.1012 7.6009 0.0
+2.8745 6.0817 0.0
+1.7358 5.4503 0.0
+```
+
+
+This exmaple uses the ```DataBufferFromCallback``` to get the training data and configuration model is in ```res/LogisticRegression.ini```
+
+```
+# Network Section : Network
```
Comment (zhoonit) [inline Applications/LogisticRegression/README.md:27]: Could you `s/Network/Model` here and for the given ini file?

**Review by zhoonit — APPROVED** [review]
LGTM

**Review by kparichay — APPROVED** [review]
LGTM

File: Applications/LogisticRegression/README.md (hunk 1 above)
Comment (jijoongmoon) [reply Applications/LogisticRegression/README.md]: fixed.

**Review by zhoonit — APPROVED** [review]
LGTM

**Pull request merged**